app_check/src/desktop/debug_provider_desktop.cc

```diff
@@ -90,9 +92,12 @@ void DebugAppCheckProvider::GetToken(
     std::function<void(AppCheckToken, int, const std::string&)>
         completion_callback) {
   // Identify the user's debug token
-  // TODO(amaurice): For now uses an environment variable, but should use other
-  // options.
-  const char* debug_token = std::getenv("APP_CHECK_DEBUG_TOKEN");
+  const char* debug_token;
```

Comment:
nit debug_token_ptr?

Reply:
Changed it to debug_token_cstr, to make it more obvious what it is.